benchmarks/pubmedqa/pubmedqa.py

```diff
@@ -0,0 +1,126 @@
+"""
+
+
+# eval pubmedqa/pubmedqa.py
+# inspect eval pubmedqa.py
+"""
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.scorer import choice
+from inspect_ai.solver import multiple_choice
+
+"""
+qiaojin/PubMedQA
```

Comment:
Once you've got everyting working as you want I think we should remove this dataset structure comment.